README.md

```diff
@@ -37,19 +37,74 @@ git clone https://github.com/PetJournal/petjournal.api.git
 docker compose -f docker-compose.dev.yml up -d
```

Comment:
I suggest creating a `docker:up` command, for example, to abstract this command.

Reply:
I agree, good suggestion.